contrib/fuzz/archive_fuzzer.go

```diff
@@ -67,7 +67,7 @@ func FuzzApply(data []byte) int {
 // that targets archive.ImportIndex()
 func FuzzImportIndex(data []byte) int {
 	f := fuzz.NewConsumer(data)
-	tarBytes, err := f.TarBytes()
+	tarBytes, err := f.GetBytes()
```

Comment:
Does it return arbitrary bytes or bytes that are valid as a tar archive?

Reply:
arbitrary bytes. It should work better with new bug detectors.